code-challenges.test.js

```diff
@@ -69,3 +126,4 @@ var hand3 = [5, 5, 5, 5, 4]
 
 
 // b) Create the function that makes the test pass.
+//not even sure where to start with this one 
```

Comment:
I'd like to see an attempt at writing a function. Even one that takes in hand and does something silly with it. Or even some pseudo code trying to break the problem down. 

But since we've been over this in class I am going to ask that you finish this problem and make it work before I approve.